Express/model.js

```diff
@@ -36,7 +36,33 @@ const fetchArticles = async () => {
                                             comments.article_id = articles.article_id
                                    GROUP BY articles.article_id
                                    ORDER BY articles.created_at DESC;
-  `);
+    `);
   return rows;
 };
-module.exports = { fetchAllTopics, fetchArticleByID, fetchArticles };
+
+const fetchComments = async (article_id) => {
+  const { rows } = await db.query(
+    `select comments.comment_id,
+                comments.votes,
+                comments.created_at,
+                comments.author,
+                comments.body,
+                comments.article_id
+         from comments
+                  join articles on articles.article_id = comments.article_id
+         where articles.article_id = $1
+         group by comment_id`,
+    [article_id],
+  );
+  if (rows.length === 0) {
```

Comment:
fetching comments by article id, if there are no comments for that particular article shouldn't generate an error, this is something that should be covered in advanced error handling, but feel free to read the notes if you want to check it out earlier.